docs/e2e-tests.md

```diff
@@ -72,6 +72,24 @@ Add the accessibility-id to a React component using `testID('Example-Component')
 
 For more complex queries you should add/use a custom [Selector]('../e2e/native/Selector.ts) using [predicate strings](https://github.com/facebookarchive/WebDriverAgent/wiki/Predicate-Queries-Construction-Rules) for iOS and [UiSelectors](https://developer.android.com/reference/androidx/test/uiautomator/UiSelector) for Android.
 
+#### Using Appium
```

Comment:
I think this belongs to `native/docs/e2e-testing`. There is also already a section about Appiu .

Reply:
Good point. Moved it.